web/packages/teleterm/src/mainProcess/rootClusterProxyHostAllowList.ts

```diff
@@ -47,17 +48,19 @@ export function manageRootClusterProxyHostAllowList({
   logger: Logger;
   allowList: RootClusterProxyHostAllowList;
 }) {
-  let abortController: tshd.TshAbortController;
+  let abortController: AbortController;
 
   const refreshAllowList = async () => {
     // Allow only one call to be in progress. This ensures that on subsequent calls to
     // refreshAllowList, we always store only the most recent version of the list.
     abortController?.abort();
-    abortController = tshdClient.createAbortController();
+    abortController = new AbortController();
 
     let rootClusters: tshd.Cluster[];
     try {
-      rootClusters = await tshdClient.listRootClusters(abortController.signal);
+      rootClusters = await tshdClient.listRootClusters(
+        objectifyAbortSignal(abortController.signal)
+      );
```

Comment:
Actually, we have 7 callsites that need to import `objectifyAbortSignal` (and 5 of them use their own abort controller). Also, eventually we want to remove the `tshdClient` layer, so we will still need to import it from somewhere.
Because of that, I'm leaning towards importing it from a separate file.